### DISCUSSION TITLE
Transferring Licence from Chrome to Edge

### DISCUSSION THREAD

**Discussion by Meronique** [aclap-dev/video-downloadhelper#26] [Help and Q&A]
### Your question

Hey. I'm switching browsers from Chrome to Edge. Is there a way to transfer my paid licence?

**Comment by GrampaWildWilly** [comment]
No.  You have to buy a new one.  Depending on how recently you bought your existing license (within the past month) @jcfrog will be able to refund it.  Full information on the VDH license is here:

https://groups.google.com/g/video-downloadhelper-q-and-a/c/12-fRkVJ8qo

You need to read that as well as all the threads it refers to.
